app/Http/Controllers/Manager/StatsController.php

```diff
@@ -324,4 +319,35 @@ private function getSiteIdFromRequest(Request $request): ?string
 
         return $site->uuid;
     }
+
+    private function callbackTransformingId(): Closure
+    {
+        return function ($item) {
+            return $this->transformPublicIdToPrivateId($item);
+        };
+    }
+
+    private function callbackFilteringNullFromAdvertiserStats(): Closure
+    {
+        return function (array $item) {
+            if ((array_key_exists('campaignId', $item) && is_null($item['campaignId']))
+                || (array_key_exists('bannerId', $item) && is_null($item['bannerId']))) {
+                return false;
+            }
+
+            return true;
+        };
+    }
```

Comment:
PHP `isset` function checks if a key exists and if a value is different than null ;-)

```
if (!isset($item['campaignId']) || !isset($item['bannerId']) {
    return false;
}

return true;
```